pkg/apis/core/v1/helper/helpers.go

```diff
@@ -85,6 +85,21 @@ func HugePageSizeFromResourceName(name v1.ResourceName) (resource.Quantity, erro
 	return resource.ParseQuantity(pageSize)
 }
 
+// HugePageUnitSizeFromByteSize returns hugepage size has the format.
+// `size` must be guaranteed to divisible into the largest units that can be expressed.
```

Comment:
if size does not match the documented requirements, I would have expected an error... it's unclear what is returned in error cases or how a caller would know they had provided invalid data

Reply:
I will update error handling about size requirements.

Reply:
updated it.